docs/setup.md

```diff
@@ -0,0 +1,18 @@
+| [Home](../README.md) |
+|--------------------------------------------|
+
+# Installation
```

Comment:
Make it as per standard refer to https://github.com/fortinet-fortisoar/solution-pack-phishing-email-response/blob/develop/docs/setup.md